Pipfile

```diff
@@ -0,0 +1,48 @@
+[[source]]
+name = "pypi"
+url = "https://pypi.org/simple"
+verify_ssl = true
+
+[dev-packages]
+black = "*"
+boto3-stubs = "*"
+coveralls = "*"
+Django = "<4"
+django-stubs = "*"
+flake8 = "*"
+Flask = "*"
+isort = "*"
+mock = "*"
+mypy = "*"
+nose = "*"
+nose-timer = "*"
+pipenv = "*"
+placebo = "<0.10"
```

Comment:
duplicate of requirement in [packages] below

Reply:
Yep good call - this was a little confusing because `placebo` _should_ live in `dev-packages`, but we need it to live in `packages` temporarily until we're able to unpin its version.  I removed it here and left a comment by the entry in `packages` to remind us to move it to `dev-packages` once the version is unpinned.